homeassistant/components/backup/manager.py

```diff
@@ -312,6 +312,10 @@ async def _add_platform(
         """Add a backup platform manager."""
         self._add_platform_pre_post_handler(integration_domain, platform)
         await self._async_add_platform_agents(integration_domain, platform)
+        LOGGER.debug("Backup platform %s loaded", integration_domain)
+        LOGGER.debug("%s platforms loaded in total", len(self.platforms))
+        LOGGER.debug("%s agents loaded in total", len(self.backup_agents))
+        LOGGER.debug("%s local agents loaded in total", len(self.local_backup_agents))
```

Comment:
This change seems unrelated to this PR?

Reply:
Yeah, I added it when troubleshooting the test I was changing. It helped writing the test.

Reply:
The debug logging we already had during setup doesn't capture platforms that are loaded later.